x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/view/details/components/activity_log_date_range_picker/index.tsx

```diff
@@ -8,116 +8,135 @@
 import { useDispatch } from 'react-redux';
 import React, { memo, useCallback } from 'react';
 import styled from 'styled-components';
-import moment from 'moment';
-import { EuiFlexGroup, EuiFlexItem, EuiDatePicker, EuiDatePickerRange } from '@elastic/eui';
+import dateMath from '@elastic/datemath';
+import {
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiSuperDatePicker,
+  EuiSuperDatePickerRecentRange,
+} from '@elastic/eui';
 
-import * as i18 from '../../../translations';
 import { useEndpointSelector } from '../../../hooks';
 import { getActivityLogDataPaging } from '../../../../store/selectors';
+import { DEFAULT_TIMEPICKER_QUICK_RANGES } from '../../../../../../../../common/constants';
+import { useUiSetting$ } from '../../../../../../../../public/common/lib/kibana';
```

Comment:
I see `/public/` in this path - where is this being imported from? 
if from security solution, then you might have one too many `../` here

Reply:
Indeed. It's being imported from `security_solution/public`

Reply:
fixed a40c21f113763c1070f1d6ca24100575ac4a84da